src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.Tag;
+
+/**
+ * Association class between Person and Tag. Each tagging object represents a
+ * tagging action taken for a person, that is, either deletion or addition of a
+ * tag, during the program session.
+ *
+ */
+
+public class Tagging {
+
+    public static final String addedSign = "+";
+    public static final String deletedSign = "-";
+
+    private boolean isForDeletedTag; // else the tag is added
+    private Tag tag;
+    private ReadOnlyPerson taggedPerson;
+
+    public Tagging(ReadOnlyPerson taggedPerson, Tag tag, boolean isForDeletedTag) {
```

Comment:
As discussed in tutorial, what if we want to support a third operation like `edit`? Your boolean value can only handle two cases.